domino/domino.py

```diff
@@ -349,6 +350,41 @@ def model_version_publish(self, model_id, file, function, environment_id,
         response = requests.post(url, auth=('', self._api_key), json=request)
         return response.json()
 
+    # Datasets Functions
+    def datasets_list(self, projectId=None):
+        self.requires_at_least("3.6.0")
+        url = self._routes.datasets_list(projectId)
+        return self._get(url)
+
+    def datasets_create(self, projectId, datasetName, datasetDescription): 
+        self.requires_at_least("3.6.0")
+        url = self._routes.datasets_create()
+        request = {
+            "datasetName": datasetName,
+            "description": datasetDescription,
+            "projectId": self._project_id
+        }
+        response = requests.post(url, auth=('', self._api_key), json=request)
+        return response.json()
+
+    def datasets_details(self, datasetId): 
+        self.requires_at_least("3.6.0")
+        url = self._routes.datasets_details(datasetId)
+        return self._get(url)
+
+    def datasets_update_details(self, datasetId, datasetName, datasetDescription):
+        self.requires_at_least("3.6.0")
+        url = self._routes.datasets_details(datasetId, datasetName, datasetDescription)
+        request = {
+            "datasetName": datasetName,
+            "description": datasetDescription
+        }
+        print(url)
+        response = requests.patch(url, data=request, auth=('', self._api_key))
```

Comment:
I tried this function `datasets_update_details`, it didn't worked for me. It throwed this error 
```
>>> qs.datasets_update_details("5e7db98a9870cb46291f4305", "new_winds", "new description")
https://maksim160405.deploys-kappa.domino.tech/dataset/5e7db98a9870cb46291f4305
<PreparedRequest [PATCH]>
Unsupported Media Type
<Response [415]>
 ```

Changing `data` parameter in `requests.patch` to `json` works.
```
response = requests.patch(url, json=request, auth=('', self._api_key))
```
 Can you have another look on it. Thanks